testsuite/objects/__init__.py

```diff
@@ -97,6 +97,18 @@ def asdict(self):
         return {"valueFrom": {"authJSON": self.authJSON}}
 
 
+@dataclass
+class Property:
+    """Dataclass for static and dynamic values. Property is a Value with name."""
+
+    name: str
+    value: ABCValue
+
+    def asdict(self):
+        """Override `asdict` function"""
+        return {"name": self.name, **asdict(self.value)}
```

Comment:
this is not necessary with custom asdict function anymore, is it?

Reply:
@mganisin another possibility how to implement these properties - https://github.com/Kuadrant/testsuite/pull/212/files#diff-25923b13ccdf4676496d71da370f7c9ccd254043ebe12c08c0faa40e7e32be2cR74-R86